js/app.js

```diff
@@ -39,7 +39,7 @@ angular.module('PathOfDamage', [])
   };
 
   $scope.updateDamageValues = function (skipSerialization) {
-    $scope.hits = $scope.hits.map(function (hit) { return hit.hit ? calcDamage(hit.hit) : null });
+    $scope.hits = $scope.hits.map(function (hit) { return hit && hit.hit ? calcDamage(hit.hit) : null });
```

Comment:
This didn't need to be handled before my changes, I think, but it prevents errors anyway now